.github/workflows/build.yml

```diff
@@ -65,11 +65,12 @@ jobs:
       - name: "Verify no plugin issues"
         run: mvn artifact:check-buildplan --batch-mode --no-transfer-progress
 
-      - name: "Verify reproducible build"
+      - name: "Do clean install of dependencies"
         # See https://maven.apache.org/guides/mini/guide-reproducible-builds.html#how-to-test-my-maven-build-reproducibility
```

Comment:
Might be good to move this `# See ...` comment above the `name: "Do clean ...` since it applies to both steps now and not just the `mvn clean install`.

Reply:
Ah oke yes makes sense. I've updated this.